lib/Echidna/Solidity.hs

```diff
@@ -107,3 +115,16 @@ loadSolidity fp name = let ensure (l, e) = if null l then throwM e else pure ()
     case find (not . null . snd) tests of
       (Just (t,_)) -> throwM $ TestArgsFound t
       Nothing      -> return (loaded, funs, fst <$> tests)
+
+-- | Basically loadSolidity, but prepares the results to be passed directly into
+-- a testing function.
+loadTesting :: (MonadIO m, MonadThrow m, MonadReader x m, Has SolConf x)
```

Comment:
Can we rename for clarity? Maybe `loadSolTests` or something

Reply:
done